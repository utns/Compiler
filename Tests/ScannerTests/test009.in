90{qwerty}23