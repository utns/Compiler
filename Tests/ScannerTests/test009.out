1    1   integer number  90                   90
1    11  integer number  23                   23
